src/borg/testsuite/archiver.py

```diff
@@ -4199,6 +4199,42 @@ def test_sort_option(self):
 
         assert all(x in line for x, line in zip(expected, output.splitlines()))
 
+    def test_format_option(self):
+        self.cmd('init', '--encryption=repokey', self.repository_location)
+
+        self.create_regular_file('a_file', size=8)
+        test_archive0 = self.repository_location + '::test0'
+        self.cmd('create', test_archive0, "input")#self.input_path)
```

Comment:
^ looks strange

Reply:
Do you mean the `#self.input_path` part? I wasn't sure about what to use here and I left this comment there on purpose to get feedback...

Reply:
ah! just search the file you edited, there are lots of other such tests there.

Reply:
It is a long file... I see cases where one solution seems to be used, and cases where the other is preferred (unless I got confused).
Maybe I should call the `create_test_files` aux method? Do you think it is a good idea, instead of creating files by hand?

Reply:
create_test_files / create_regular_file add some convenience, so use them if nothing speaks against that.